src/ranged.cpp

```diff
@@ -1573,6 +1573,15 @@ dispersion_sources player::get_weapon_dispersion( const item &obj ) const
         dispersion.add_range( std::max( vol - get_skill_level( skill_driving ), 1 ) * 20 );
     }
 
+    if( get_option< bool >( "IMPROVED_RANGED_ACCURACY" ) )
+    {
+        double cbmBonus =  has_bionic( bionic_id( "bio_targeting" ) ) ? 2 : 0;
+        double skillEffect = 1 - std::min( ( double( get_skill_level( skill_gun ) + double( get_skill_level(
+            obj.gun_skill() ) ) ) ) / 2 + cbmBonus, double( MAX_SKILL ) ) * 0.0833;
```

Comment:
The formula is too hard to understand imo (with all these lots of parentheses). Maybe move the first parameter of `std::min` into its own variable, like `cbmBonus` above?

Also, where does this 0.0833 magic number came from?

Reply:
I dunno, I could use a calculator and follow the formula just fine. The closer to 10/10 skill level I inputted the closer to .0833 I got.  I don't know what the result of the formula equates to when multiplied by the dispersion.add_multiplier gets affected. On the forums he stated that the dispersion total can end up lower than the hard cap of 45.